multiqc.wdl

```diff
@@ -0,0 +1,85 @@
+version 1.0
+
+task MultiQC {
+    input {
+        String? preCommand
+        File analysisDirectory
+        Array[File] dependencies   # This must be used in order to run multiqc after these tasks.
+        Boolean force = false
+        Boolean dirs = false
+        Int? dirsDepth
+        Boolean fullNames = false
+        String? title
+        String? comment
+        String? fileName
+        String? outDir
+        String? template
+        String? tag
+        String? ignore
+        String? ignoreSamples
+        Boolean ignoreSymlinks = false
+        File? sampleNames
+        File? fileList
+        Array[String]+? exclude
+        Array[String]+? module
+        Boolean dataDir = false
+        Boolean noDataDir = false
+        String? dataFormat
+        Boolean zipDataDir = false
+        Boolean export = false
+        Boolean flat = false
+        Boolean interactive = false
+        Boolean lint = false
+        Boolean pdf = false
+        Boolean megaQCUpload = false # This must be actively enabled in my opinion. The tools default is to upload.
+        File? config  # A directory
+        String? clConfig
+        Boolean verbose  = false
+        Boolean quiet = false
+    }
+
+    String outputDir = if defined(outDir) then select_first([outDir]) else "."
```

Comment:
Maybe it's better to make `outDir` mandatory?

Reply:
Still, it seems somewhat dangerous to default to the current working directory, but I guess since this is just a report it's okay.

Either way, this line can be removed if you change the input to:
`String outDir = "."`

Reply:
Agree